browser/brave_ads/ads_tab_helper.h

```diff
@@ -13,11 +13,14 @@
 
 #include "base/memory/raw_ptr.h"
 #include "base/memory/weak_ptr.h"
+#include "brave/browser/brave_ads/search_result_ad/search_result_ad_handler.h"
 #include "build/build_config.h"
 #include "components/sessions/core/session_id.h"
 #include "content/public/browser/media_player_id.h"
 #include "content/public/browser/web_contents_observer.h"
 #include "content/public/browser/web_contents_user_data.h"
+#include "mojo/public/cpp/bindings/remote.h"
```

Comment:
these don't appear to be needed in the ads_tab_helper header file

Reply:
forgot to remove, thanks!